Dockerfile

```diff
@@ -1,6 +1,6 @@
-ARG DEBIAN_VERSION=11.9
+ARG DEBIAN_VERSION=12.7
```

Comment:
Do we really need to replicate this as a default value?

Reply:
Never mind. I thought this was in the base image Dockerfile, but it is not. It will be harder to make both of them include the same definition indeed.